crates/re_query/tests/query_tests.rs

```diff
@@ -175,7 +175,7 @@ fn no_instance_join_query() {
     // │ u64      ┆ struct[2] ┆ u32        │
     // ╞══════════╪═══════════╪════════════╡
     // │ 0        ┆ {1.0,2.0} ┆ 4278190080 │
-    // ├╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌┤
+    // ├╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌┤
```

Comment:
?

Reply:
There was one too many. Looks nicer now ;)

Reply:
Hm okay that's only my editor that doensn't display ┼ correctly -.-